### PR TITLE
{bio} [goolf/1.7.20] Discovar / DiscovarDenovo

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 19 (4 easyconfigs in this PR)
dell.lan - Linux fedora 27, Intel(R) Core(TM) i7-4600U CPU @ 2.10GHz, Python 2.7.14
See https://gist.github.com/437e252b0807e5193c7688b42c7a3a6b for a full test report.

**Comment by verdurin** [comment]
(Ignore that failed test build)

File: easybuild/easyconfigs/j/jemalloc/jemalloc-4.2.0-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+version = '4.2.0'
+
+homepage = 'http://www.canonware.com/jemalloc'
+description = """jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and
+ scalable concurrency support."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://github.com/jemalloc/jemalloc/releases/download/%(version)s/']
+sources = ['%(name)s-%(version)s.tar.bz2']
```
Comment (boegel) [inline easybuild/easyconfigs/j/jemalloc/jemalloc-4.2.0-goolf-1.7.20.eb]: @pescobar missing `SHA256` checksum

File: easybuild/easyconfigs/s/SAMtools/SAMtools-1.3.1-goolf-1.7.20.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+homepage = 'http://www.htslib.org/'
+description = """SAM Tools provide various utilities for manipulating alignments in the SAM format,
+ including sorting, merging, indexing and generating alignments in a per-position format."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['https://github.com/samtools/%(namelower)s/releases/download/%(version)s']
+sources = [SOURCELOWER_TAR_BZ2]
```
Comment (boegel) [inline easybuild/easyconfigs/s/SAMtools/SAMtools-1.3.1-goolf-1.7.20.eb]: @pescobar missing `SHA256` checksums (also for `patches`), use `eb --inject-checksums`

File: easybuild/easyconfigs/s/SAMtools/SAMtools-1.3.1-goolf-1.7.20.eb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+##
+name = 'SAMtools'
+version = '1.3.1'
+
+homepage = 'http://www.htslib.org/'
+description = """SAM Tools provide various utilities for manipulating alignments in the SAM format,
+ including sorting, merging, indexing and generating alignments in a per-position format."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'optarch': True, 'pic': True}
```
Comment (boegel) [inline easybuild/easyconfigs/s/SAMtools/SAMtools-1.3.1-goolf-1.7.20.eb]: @pescobar Please drop `'optarch': True`, since that's the default.

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): uses a deprecated toolchain.
For more information on deprecated toolchains, see https://easybuild.readthedocs.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains.
Please consider reopening or submitting a new PR using a supported toolchain.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**